source/includes/service.md

```diff
@@ -125,7 +125,7 @@
         {"name": "tagthree3"}
   ],
   "target_num_containers": 2,
-
+  "autoredeploy": False,
```

Comment:
this is in the wrong place (has to be ordered alphabetically), and `False` is not a valid JSON data type (it must be `false`).